src/model.js

```diff
@@ -363,7 +347,7 @@ var FlatsealModel = GObject.registerClass({
             const appId = GLib.path_get_basename(file.get_path());
             const activePath = GLib.build_filenamev([file.get_path(), 'current', 'active']);
 
-            if (GLib.access(activePath, 0) === 0 && _blacklist.includes(appId) == false)
+            if (GLib.access(activePath, 0) === 0 && appId.endsWith(".BaseApp") == false)
```

Comment:
Can you:
1. Invert the order? Check for `.BaseApp` first. One disk access less ;)
2. Use single quotes.
3. Add a `/* XXX will only work with app-ids that follow the BaseApp convention */` (or whatever message you seem fit).